project/AppDependencies.scala

```diff
@@ -2,20 +2,20 @@ import sbt.*
 
 object AppDependencies {
 
-  private lazy val mongoHmrcVersion     = "1.7.0"
-  private lazy val bootstrapPlayVersion = "8.4.0"
+  private lazy val mongoHmrcVersion     = "2.0.0"
+  private lazy val bootstrapPlayVersion = "8.6.0"
 
   private lazy val compile: Seq[ModuleID] = Seq(
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-play-30"         % mongoHmrcVersion,
     "uk.gov.hmrc"       %% "bootstrap-frontend-play-30" % bootstrapPlayVersion,
-    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "8.5.0"
+    "uk.gov.hmrc"       %% "play-frontend-hmrc-play-30" % "10.1.0"
   )
 
   private lazy val test: Seq[ModuleID] = Seq(
     "uk.gov.hmrc.mongo" %% "hmrc-mongo-test-play-30" % mongoHmrcVersion,
     "uk.gov.hmrc"       %% "bootstrap-test-play-30"  % bootstrapPlayVersion,
     "org.jsoup"          % "jsoup"                   % "1.17.2",
-    "org.mockito"       %% "mockito-scala-scalatest" % "1.17.30",
+    "org.mockito"       %% "mockito-scala-scalatest" % "1.17.31",
```

Comment:
Can you try to remove this as discussed in tech session to use `org.mockito.Mockito` and `org.mockito.ArgumentMatchers` instead of the `org.mockito.xxxSugar`, thanks. An example of the changes can be seen [here](https://github.com/hmrc/pbik-frontend/pull/254/files). If it is a lot of changes, you can ignore this.